src/main/java/com/faforever/api/clan/ClansController.java

```diff
@@ -0,0 +1,81 @@
+package com.faforever.api.clan;
+
+import com.faforever.api.data.domain.Clan;
+import com.faforever.api.data.domain.Player;
+import com.faforever.api.player.PlayerRepository;
+import com.faforever.api.utils.AuthenticationHelper;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.Authentication;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+
+@RestController
+@RequestMapping(path = "/clan")
+public class ClansController {
```

Comment:
Clans* or Clan*? `/clan` or `/clans`?
(Tip: keep it consistent with existing code)